src/cli/utils/sharepoint.go

```diff
@@ -9,12 +9,16 @@ import (
 const (
 	LibraryItemFN = "library-item"
 	LibraryFN     = "library"
+	ListsItemFN   = "lists-item"
+	ListsFN       = "lists"
```

Comment:
The plural here is a little confusing.  Let's stick with just `list` instead.
```suggestion
	ListItemFN   = "list-item"
	ListFN       = "list"
```